WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionSummaryViewModel.cs

```diff
@@ -59,8 +62,11 @@ public void UpdateTransaction(BuildTransactionResult transactionResult)
 			destinationAmount.GenerateFiatText(_wallet.Synchronizer.UsdExchangeRate, "USD");
 		AmountText = $"{btcAmountText}{fiatAmountText}";
 
+		var feeDisplayFormat = (FeeDisplayFormat?)Services.UiConfig.FeeDisplayFormat ?? FeeDisplayFormat.BTC;
+		var btcFeeMoneyUnit = feeDisplayFormat.ToMoneyUnit();
```

Comment:
nit: You name a similar variable `feeMoneyUnit`, so IMO we should be consistent whenever possible.